documentation/docs/reference/services/Upload.md

```diff
@@ -40,6 +40,45 @@ Response format:
 }
 ```
 
+GET /upload/photo/USERID/
+--------------------------
+
+Returns the photo stored for the user with the `id` `USERID`.
+
+Response format:
+```
+{
+	"id": "github0000001",
+	"photo": "https://bucket.s3.amazonaws.com/photo"
+}
+```
+
+GET /upload/photo/
+-------------------
+
+Returns the photo stored for the user with the `id` stored in the given JWT in the Authorization header.
+
+Response format:
+```
+{
+	"id": "github0000001",
+	"photo": "https://bucket.s3.amazonaws.com/photo"
+}
+```
+
+GET /upload/photo/upload/
```

Comment:
Reading the service code below, I understand better - this service is supposed to upload photos and return the link.
Following earlier convention (e.g. our other update information endpoints), this should be changed to a PUT endpoint - `PUT /upload/photo`.

Reply:
The endpoint is returning a presigned s3 url which will accept a PUT request for an image. The request to the API should be a GET request.

Reply:
I was thinking of renaming these to `GET /upload/resume/url/` and `GET /upload/photo/url/` but that change would break the application, so I was going to wait until the application closes.